.ci/.jenkins-schema.yml

```diff
@@ -0,0 +1,17 @@
+---
+agents:
+  - REPO: "apm-agent-dotnet"
+    SPEC_FILEPATH: "test/Elastic.Apm.Tests/TestResources/json-spec"
```

Comment:
@gregkalapos and @russcam, what folder then?
- https://github.com/elastic/apm-agent-dotnet/tree/master/test/Elastic.Apm.Tests/TestResources/
- https://github.com/elastic/apm-agent-dotnet/tree/master/src/Elastic.Apm.Specification

Reply:
A `specs` directory in https://github.com/elastic/apm-agent-dotnet/tree/master/src/Elastic.Apm.Specification will be great, thanks!

Reply:
```suggestion
  - REPO: "apm-agent-dotnet"
    SPEC_FILEPATH: "src/Elastic.Apm.Specification/specs"
```